javamelody-core/src/main/java/net/bull/javamelody/internal/model/CounterRequestContext.java

```diff
@@ -93,6 +94,7 @@ private CounterRequestContext(Counter parentCounter, CounterRequestContext paren
 		this.remoteUser = remoteUser;
 		this.threadId = threadId;
 		this.startTime = startTime;
+		this.currentTime = System.currentTimeMillis() - startTime;

```

Comment:
currentTime is always 0, because startTime = System.currentTimeMillis() just above.

Reply:
I suppose that `getDuration(long)` should be used instead of `getCurrentTime()`